stdlib/2/__builtin__.pyi

```diff
@@ -1438,7 +1438,7 @@ class BaseException(object):
         __cause__: Optional[BaseException]
         __context__: Optional[BaseException]
         __traceback__: Optional[TracebackType]
-    def __init__(self, *args: object) -> None: ...
+    def __init__(self, *args: object, **kwargs) -> None: ...
```

Comment:
Can you add `: object` as the type? We like to keep all arguments typed.

Reply:
Makes sense, thanks.